### ISSUE TITLE
Will not save reminder

### ISSUE THREAD

**Issue by TiredTyler53** [Acidham/alfred-apple-reminder#4]
I can input all variables but it will not save it in the app.
I believe there is something wrong with the AppleScript but I am not sure?

[15:21:45.810] Logging Started...
[15:22:05.432] Apple Reminders[Keyword] Processing complete
[15:22:05.438] Apple Reminders[Keyword] Passing output 'Sell ATT Position' to Arg and Vars
[15:22:05.439] Apple Reminders[Arg and Vars] Processing complete
[15:22:05.440] Apple Reminders[Arg and Vars] Passing output '' to Keyword
[15:22:10.862] Apple Reminders[Keyword] Processing complete
[15:22:10.868] Apple Reminders[Keyword] Passing output '9/22' to Arg and Vars
[15:22:10.868] Apple Reminders[Arg and Vars] Processing complete
[15:22:10.869] Apple Reminders[Arg and Vars] Passing output '' to Keyword
[15:22:12.959] Apple Reminders[Keyword] Processing complete
[15:22:12.979] Apple Reminders[Keyword] Passing output '8am' to Arg and Vars
[15:22:12.980] Apple Reminders[Arg and Vars] Processing complete
[15:22:12.981] Apple Reminders[Arg and Vars] Passing output '' to Keyword
[15:22:18.713] Apple Reminders[Keyword] Processing complete
[15:22:18.725] Apple Reminders[Keyword] Passing output 'Remind Keen' to Arg and Vars
[15:22:18.726] Apple Reminders[Arg and Vars] Processing complete
[15:22:18.727] Apple Reminders[Arg and Vars] Passing output '' to Arg and Vars
[15:22:18.728] Apple Reminders[Arg and Vars] Processing complete
[15:22:18.728] Apple Reminders[Arg and Vars] Passing output 'Sell ATT Position;9/22;8am;Remind Keen' to Run NSAppleScript
[15:22:19.029] ERROR: Apple Reminders[Run NSAppleScript] {
    NSAppleScriptErrorBriefMessage = "The variable DueDate is not defined.";
    NSAppleScriptErrorMessage = "The variable DueDate is not defined.";
    NSAppleScriptErrorNumber = "-2753";
    NSAppleScriptErrorRange = "NSRange: {3900, 7}";
}
[15:22:19.047] Apple Reminders[Run NSAppleScript] Processing complete
[15:22:19.048] Apple Reminders[Run NSAppleScript] Passing output '' to Post Notification

**Comment by Acidham** [comment]
can you please try use "tomorrow" as due date.

if successful try with "09/22/21"

**Comment by TiredTyler53** [comment]
Yes I also deleted and redownloaded it into workflow. This is now working as usual. Can you save to other lists? It automatically sends it to the first list onlyBest Wishes,

J. Tyler Hartrim
832-879-6658On Sep 21, 2021, at 3:29 PM, Acidham ***@***.***> wrote:can you please try use "tomorrow" as due date.if successful try with "09/22/21"—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Acidham** [comment]
Yep in the last step you can type target list.

**Comment by TiredTyler53** [comment]
Okay, yes I can type in another list but it always saves it to the first list in the row. For example, I have 2 lists Remind me and to do. Even if I type To Do it will save it in the remind me list because it is the first one.Best Wishes,

J. Tyler Hartrim
832-879-6658On Sep 22, 2021, at 4:11 PM, Acidham ***@***.***> wrote:Yep in the last step you can type target list.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.

**Comment by Acidham** [comment]
Ah, I see... My lists are all without whitespace in between, and it works. As soon as I create a "to do" list, it does not work. Seems my script cannot find it. 
But before investigating further, can you please confirm with list without white spaces in between?

**Comment by Acidham** [comment]
Found the issue and fixed with https://github.com/Acidham/alfred-apple-reminder/releases/tag/v2.2.1
